third_party/silabs/efr32_sdk.gni

```diff
@@ -249,11 +249,20 @@ template("efr32_sdk") {
         # network_manager component
         "${wifi_sdk_root}/components/service/network_manager/inc",
 
+        # trng component
+        "${wifi_sdk_root}/components/device/silabs/si91x/wireless/crypto/inc",
+        "${wifi_sdk_root}/components/device/silabs/si91x/wireless/crypto/trng/inc",
+
         "${wifi_sdk_root}/components/protocol/wifi/si91x",
       ]
     }
     if (use_rs9116 || use_SiWx917) {
-      _include_dirs += [ "${chip_root}/src/platform/silabs/rs911x" ]
+      _include_dirs += [
```

Comment:
when it was added to the template from the app build.gn we add this todo
`# TODO efr32_sdk should not need a header from this location`
Maybe add this todo back. I wonder why SDK files needs includes from example codes

Reply:
The reason is that here, efr32_sdk is not a separate compilation unit, like a library. The efr32_sdk is a template, which results in a bundle of configuration and files that are merged with the application. So these files are actually part of the application, which depends on the platform code. Currently, there is now avoiding it. The SDK leave holes in that must be filled by the application, thus the template.
However, I have put the comment, if just to remember these dependencies are a mess.